modules/application/application.js

```diff
@@ -1,27 +1,27 @@
 class Application {
   constructor() {
+    this.addTask = document.querySelector('.add-task');
     this.domList = document.querySelector('.todo-list');
-    this.todoList = [{ index: 0, description: 'Set up project', completed: true },
-      { index: 1, description: 'Integrate webpack', completed: true },
-      { index: 2, description: 'Create github repo', completed: true },
-      { index: 3, description: 'Implement add to list', completed: false },
-      { index: 4, description: 'Implement remove from list', completed: false },
-      { index: 5, description: 'Implement drag and drop feature', completed: false },
-      { index: 6, description: 'submit project for review', completed: false },
-    ];
+    this.todoList = [];
   }
 
   initApp = () => {
+    this.onLoadList();
+    this.updateDom();
+  }
+
+  updateDom = () => {
     const ref = this;
+    this.domList.innerHTML = '';
     this.todoList.forEach((element) => {
       let completed = '';
       if (element.completed) {
         completed = 'done-task';
       }
       ref.domList.innerHTML = `${ref.domList.innerHTML} <li class="todo-item">
       <div class="checker"><span class=""><input class="list-check-${element.index}" type="checkbox"></span></div>
-      <span class="${completed}">${element.description}</span>
-      <i class="fa fa-trash-o float-right"></i>
+      <span class="${completed} desc" contentEditable="true">${element.description}</span>
+      <i class="fa fa-trash-o float-right delete"></i>
```

Comment:
The application is NOT performing as expected when I tried to edit a task. After editing a task, when I hit enter to save it keeps adding `<br>` to the task. Kindly work on this. Please see the screenshot below.

![image](https://user-images.githubusercontent.com/10332499/154390961-5ae3e7e9-7130-4f0a-93f0-9aa956d15577.png)

Reply:
you don't hit enter to save an edited task. I put that in my pull request description.

Reply:
It would be better if you make the use `innerText` rather than an `innerHTML`

Reply:
Yeah, True. Thanks for the hint